ros2_control_demo_bringup/launch/rrbot_system_position_only_gazebo.launch.py

```diff
@@ -38,10 +40,18 @@ def generate_launch_description():
     declared_arguments.append(
         DeclareLaunchArgument(
             "use_fake_hardware",
+            default_value="false",
```

Comment:
IMO we should completely remove `use_fake_hardware` from the *gazebo* launch file. We can still explicitly pass `"use_fake_hardware": False,` to the xacro file as there shouldn't be fake hardware present with gazebo...

Reply:
Fixed with fd66e4d

Reply:
thanks!